src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/BooleanExpressionComplexityCheck.java

```diff
@@ -30,8 +30,8 @@
 
 /**
  * <p>
- * Restrict the number of {@code &amp;&amp;}, {@code ||}, {@code &amp;}, {@code |} and
- * {@code ^} in an expression.
+ * Restricts nested boolean operators ({@code &amp;&amp;}, {@code ||}, {@code &amp;}, {@code |} and
+ * {@code ^}) in an expression to a specified depth.
```

Comment:
I don't see any depth check in the check. It looks like it is just counting how many are in 1 condition and giving a violation if it reaches a certain amount.

Reply:
removed

Reply:
Depth was removed but without the wording that we are looking for a specific count, description seems like it is talking about all operators.

Reply:
Reverted back to
`Restricts the number of boolean operators ({@code &amp;&amp;}, {@code ||}, {@code &amp;}, {@code |} and {@code ^}) in an expression.`